app_test.go

```diff
@@ -31,7 +32,7 @@ func TestGetDefinitionFilePath(t *testing.T) {
 	}
 
 	for _, ver := range vers {
-		_, err := getDefinitionFilePath(ver)
+		_, err := getFilePath(ver, "definitions.yaml")
```

Comment:
Rename test function to `TestGetFilePath` so it matches the name of the function being tested